frontend/src/routes/_app/chat/$conversationId.tsx

```diff
@@ -0,0 +1,6 @@
+import { createFileRoute } from "@tanstack/react-router";
+import { Conversation } from "@components/Conversation/Conversation";
+
+export const Route = createFileRoute("/_app/chat/$conversationId")({
```

Comment:
These declarations are really ugly, do we have to do them? Why can't we define things in one place and just specify the component there? 

Taking me back to old Python where we had to add __init__.pys everywhere brainlessly for reasons we don't understand.

Reply:
I didn't really get your comment, do you mean it would be better if we had the components inside the route files?

Reply:
Like all this is doing is: {"route": <component>}.

I don't get why we have to create N files instead of one list of length N in a single file. The old version was super readable, this is not.

Reply:
Will look into code based routing tomorrow.

Reply:
Cause they're like 5 routes in one files vs 1 route in 5 files, much more readable